.gitignore

```diff
@@ -1,4 +1,5 @@
-node_modules
+/node_modules
+/packages/*/**/node_modules
```

Comment:
Cool. Much better. I thought it would not ignore "node_modules" behind. But works fine.